src/undo.h

```diff
@@ -39,6 +50,11 @@ class CTxInUndo
         if (nHeight > 0)
             ::Serialize(s, VARINT(this->nVersion), nType, nVersion);
         ::Serialize(s, CTxOutCompressor(REF(txout)), nType, nVersion);
+
+        if ((this->nVersion & 0x7f) == (SC_CERT_VERSION & 0x7f)) {
```

Comment:
make this more clear by checking also nHeight > 0 
(we store that info only if coin was pruned and this is the only case of having nHeight > 0)
Serialize also the backwardTrasferStartIdx